### ISSUE TITLE
UI bug, the clock has a different background color

### ISSUE THREAD

**Issue by Pauldb8** [aunetx/blur-my-shell#4]
Hello,

Thanks for this extension, it's close to being really good.
Couple of UI bugs:
The clock on the top panel of Gnome shows a different darker background on that specific div, which ruins the blur effect unfortunately. Pic attached. Same for other artefact on top panel such as the window title.

Also on the dash-to-dock extension this works quite well but I got a weird bug where a shadow of the icon remains after I quit focus of it. It appears with the focus, that is expected, but remains after leaving the icon focus. Pic attached, in the pic my mouse is *not* hovering the chrome icon, yet it is shown as is.

I will have a look at the code as well, I'm a developer, but I don't really know how you made it work.

Anyway thanks a lot !!
![Untitled](https://user-images.githubusercontent.com/1415753/89336522-d2036680-d699-11ea-98c4-66b6b80ead15.png)
![Screenshot from 2020-08-04 21-26-58](https://user-images.githubusercontent.com/1415753/89336340-8781ea00-d699-11ea-9c9f-223920b3a64d.png)

**Comment by aunetx** [comment]
Thanks a lot!

I noticed the clock bug, and I dont think this is fixable by myself now (i guess this is the clock being updated each second or so, and the background not being - there are a lot of implementation bugs related to either ```Clutter.Actor``` or ```Shell.BlurEffect``` code in gnome-shell).

These bugs are (at least):

- blur not being updated sometimes (e.g. for shadows / animation / misc updates) (making artefacts on blur when hovering / clicking... buttons with blur behind) 
- opacity changes being... weird? when it is changed on ~```MetaBackground``` actors of overview (so in/out animations are extremmely hard or expensive to do)
- screenshots making the shell crash when some areas are blurred

So with all of these bugs, it's quite hard to do a bugless extension... or it requires a lot of cpu cycles, if the extension redraws blurred areas each frame.

I preffered to let the bugs happen so I don't forget them, and so I can use my pc with the extension activated :)

Thanks a lot for this excellent feedback, and I will try to make things change so good developpers (maybe you?) can see how these bugs can be fixed in gnome-shell's c implementation!

Thanks again :)
